### PR TITLE
[DE] Add `Help_Centre`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Help_Centre/de.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+1. Nachdem osu! abgestürzt ist, drücke `Win` + `R`, um die Ausführen-Box zu öffnen.
+2. Tippe in der Ausführen-Box `eventvwr` ein und drücke `Enter`. Das wird die Ereignisanzeige starten.
+3. Drücke in der Ereignisanzeige, auf der linken Seite, `Windows-Protokolle` und dann `Anwendung`.
+4. Drücke auf der rechten Seite auf `Aktuelles Protokoll filtern`.
+5. Stelle sicher, dass die `Fehler`-Box im nun öffnenden Fenster einen Haken hat und klicke `OK`.
+6. Drücke `Strg` + `F` und tippe osu! in die Suchleiste. Es wird das erste Absturzprotokoll von osu! finden.
+7. Gehe in den `Details`-Tab und erweitere die `System` und `EventData` durch ein Daraufklicken.
+8. Kopiere den Text dort und füge ihn in den Support-Ticket oder Forum-Beitrag.
+
+### Frame-Zeitdiagram
```
Comment (The-Last-Cookie) [inline wiki/Help_Centre/de.md:66]: I didn't found anything for `Frame Time Graph`. If you have anything, please let me know.

Comment (respektive) [reply wiki/Help_Centre/de.md:66]: I think i wouldnt translate `Frame Time` and keep it as `Frame Time Diagramm` or `Frametime-Diagramm` or something like that

File: wiki/Help_Centre/de.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+Hast du bei irgendetwas Probleme? Wir sind hier, um zu helfen! Finde einige Lösungen zu alltäglichen Problemen in der Sitebar deines Bildschirms. Wenn dein Problem hier nicht auftaucht oder du immer noch keine Lösung hast, dann sende eine E-Mail zu [support@ppy.sh](mailto:support@ppy.sh). Alternativ kannst du dein Problem auch im [Hilfeforum](https://osu.ppy.sh/community/forums/5) schildern, in dem dir jemand weiterhilft.
+
+## Abschnitte
+
+Wähle den Abschnitt, der deinem Problem am meisten ähnelt, damit du eine passende Lösung findest.
+
+| Abschnitt | Schwerpunkte |
+| :-- | :-- |
+| [Account](/wiki/Help_Centre/Account) | osu!supporter, Einschränkungen, Einloggen, Namensänderungen, Profildaten |
```
Comment (respektive) [inline wiki/Help_Centre/de.md:23]:
```suggestion
| [Account](/wiki/Help_Centre/Account) | osu!supporter, Sperrungen, Einloggen, Namensänderungen, Profildaten |
```
not sure if there is like a commonly used translation for restrictions, but this makes more sense i think

File: wiki/Help_Centre/de.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+| [Mapping und Editor](/wiki/Help_Centre/Beatmapping) | Deine Beatmaps pflegen, Beatmap-Eigentümerschaft, Beatmap-Slots |
+| [Client](/wiki/Help_Centre/Client) | Fehler und Abstürze, Spielverlauf, Verbindung, Leistung |
+| [Installation dnd Registrierung](/wiki/Help_Centre/Installation_and_registration) | Das Spiel herunterladen, einen Account erstellen |
+| [Store](/wiki/Help_Centre/Store) | Merchandise, osu!go, osu!keyboard, osu!tablet |
+| [Webseite](/wiki/Help_Centre/Website) | Benutzer blockieren, den Support kontaktieren, Seitengestaltung |
```
Comment (respektive) [inline wiki/Help_Centre/de.md:28]:
```suggestion
| [Webseite](/wiki/Help_Centre/Website) | Benutzer blockieren, den Support kontaktieren, Erscheinungsbild der Website |
```

File: wiki/Help_Centre/de.md (hunk 1 above)
Comment (TicClick) [reply wiki/Help_Centre/de.md:66]: it's an osu! feature. if it's not translated in the client, it makes sense to keep it as is

File: wiki/Help_Centre/de.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+## Helfe uns, dir zu helfen
+
+### Log-Dateien
+
+**Log-Dateien sind genaue Aufzeichnungen was der Spiel-Client zu einem bestimmten Zeitpunkt gemacht hat. Wir können sie verwenden, um dein Problem zu lösen.**
```
Comment (respektive) [inline wiki/Help_Centre/de.md:34]:
```suggestion
**Log-Dateien sind genaue Aufzeichnungen was der Spiel-Client zu einem bestimmten Zeitpunkt gemacht hat. Wir können sie verwenden, um die Ursache des Problems zu ermitteln.**
```

File: wiki/Help_Centre/de.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+Wenn ein Mitglied des Supportteams dich nach diesen Logs gefragt hat, dann mache Folgendes:
+
+1. Öffne osu!.
+2. Klicke den `Optionen`-Knopf im Hauptmenü oder drücke `Strg` + `O`.
+3. Tippe `Updatequelle` in die Schnellsuchleiste. Das wird dich zu der aktuellen Updatequelle deines Clients bringen.
+4. Versichere dich, dass es auf `Cutting Edge (Experimentell)` eingestellt ist.
+5. Klicke den Knopf zum Neustarten, der unten auf dem Bildschirm erscheint, sobald du die Updatequelle änderst.
+6. Gehe nochmal zum `Optionen`-Menü und klicke `osu!-Ordner öffnen`.
+7. Finde das `Logs`-Verzeichnis in dem Fenster, welches sich öffnet.
+8. Wähle die benötigte Log-Datei (der Support-Vertreter wird dir sagen, welche) und hänge sie an das Support-Ticket oder den Forum-Beitrag.
```
Comment (respektive) [inline wiki/Help_Centre/de.md:47]:
```suggestion
8. Wähle die benötigte Log-Datei (der Support-Ansprechpartner wird dir sagen, welche) und hänge sie an das Support-Ticket oder den Forum-Beitrag an.
```

File: wiki/Help_Centre/de.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+Wenn ein Mitglied des Supportteams dich fragt, nach Aubsturzprotokollen in der Ereignisanzeige zu suchen, dann mache Folgendes:
+
+1. Nachdem osu! abgestürzt ist, drücke `Win` + `R`, um die Ausführen-Box zu öffnen.
+2. Tippe in der Ausführen-Box `eventvwr` ein und drücke `Enter`. Das wird die Ereignisanzeige starten.
+3. Drücke in der Ereignisanzeige, auf der linken Seite, `Windows-Protokolle` und dann `Anwendung`.
+4. Drücke auf der rechten Seite auf `Aktuelles Protokoll filtern`.
+5. Stelle sicher, dass die `Fehler`-Box im nun öffnenden Fenster einen Haken hat und klicke `OK`.
+6. Drücke `Strg` + `F` und tippe osu! in die Suchleiste. Es wird das erste Absturzprotokoll von osu! finden.
+7. Gehe in den `Details`-Tab und erweitere die `System` und `EventData` durch ein Daraufklicken.
+8. Kopiere den Text dort und füge ihn in das Support-Ticket oder den Forum-Beitrag.
```
Comment (respektive) [inline wiki/Help_Centre/de.md:64]:
```suggestion
8. Kopiere den Text dort und füge ihn in das Support-Ticket oder den Forum-Beitrag ein.
```

File: wiki/Help_Centre/de.md [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+
+Wenn du mit kniffligen Leistungsproblemen in osu! konfrontiert bist, dann ist das Frame-Zeitdiagramm ein gutes Werkzeug, um das Problem manuell zu diagnostizieren und für dein Problem eine Lösung zu finden.
+
+Wenn ein Mitglied des Supportteams dich nach einem Video oder einem Bildschirmfoto mit dem Frame-Zeitdiagramm fragt, dann mache Folgendes:
+
+1. Öffne osu!.
+2. Drücke `Strg` + `F11`, um das Frame-Zeitdiagramm anzuzeigen.
+3. Löse ein Leistungsproblem aus oder warte, bis eines auftritt.
+4. Erstelle ein Bildschirmfoto durch das Drücken von `Shift` + `F12`. Das wird das Foto auf die osu!-Server hochladen und es in einem Webbrowser-Tab öffnen.
+5. Kopiere und füge den entsprechenden Link in dein Support-Ticket oder dein Forum-Beitrag.
```
Comment (respektive) [inline wiki/Help_Centre/de.md:78]:
```suggestion
5. Kopiere und füge den entsprechenden Link in dein Support-Ticket oder deinen Forum-Beitrag ein.
```

**Review by respektive — CHANGES_REQUESTED** [review]

File: wiki/Help_Centre/de.md (hunk 1 above)
Comment (respektive) [reply wiki/Help_Centre/de.md:66]: yeah, there is no translation in the client. you gotta change it in a few other places, i didn't point it out in my review 😐

File: wiki/Help_Centre/de.md (hunk 5 above)
Comment (The-Last-Cookie) [reply wiki/Help_Centre/de.md:46]: I think `an den Forum-Beitrag.` sounds a bit bette. What do you think?

File: wiki/Help_Centre/de.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+Wenn du Probleme mit osu!-Abstürzen hast und osu! dir keine Absturzprotokolle zur Verfügung stellt, dann kannst du sie nur durch die Ereignisanzeige finden.
+
+Wenn ein Mitglied des Supportteams dich fragt, nach Aubsturzprotokollen in der Ereignisanzeige zu suchen, dann mache Folgendes:
+
+1. Nachdem osu! abgestürzt ist, drücke `Win` + `R`, um die Ausführen-Box zu öffnen.
+2. Tippe in der Ausführen-Box `eventvwr` ein und drücke `Enter`. Das wird die Ereignisanzeige starten.
+3. Drücke in der Ereignisanzeige, auf der linken Seite, `Windows-Protokolle` und dann `Anwendung`.
+4. Drücke auf der rechten Seite auf `Aktuelles Protokoll filtern`.
+5. Stelle sicher, dass die `Fehler`-Box im nun öffnenden Fenster einen Haken hat und klicke `OK`.
+6. Drücke `Strg` + `F` und tippe osu! in die Suchleiste. Es wird das erste Absturzprotokoll von osu! finden.
```
Comment (TicClick) [inline wiki/Help_Centre/de.md:61]: is `Ctrl` really `Strg`?

Comment (FamousPig) [reply wiki/Help_Centre/de.md:61]: Yes. Its short for 'Steuerung', which is the translation of 'control' and is commonly used.

Comment (The-Last-Cookie) [reply wiki/Help_Centre/de.md:61]: yep, [many German keyboards](https://en.wikipedia.org/wiki/Control_key) have `Strg` there. Strg means Steuerung (or translated Control).

[Here is an example photo](https://ratgeber-finden.de/wp-content/uploads/2017/02/Computer-Tastatur-reinigen-Tipps.jpg).

**Review by TicClick — APPROVED** [review]
as they say, *mind=blown*

**Pull request merged**